mongodbatlas/resource_mongodbatlas_ldap_configuration_test.go

```diff
@@ -0,0 +1,162 @@
+package mongodbatlas
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/spf13/cast"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/acctest"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/terraform"
+	matlas "go.mongodb.org/atlas/mongodbatlas"
+)
+
+func TestAccResourceMongoDBAtlasLDAPConfiguration_basic(t *testing.T) {
```

Comment:
I think it would be ideal to do at least one test that includes a userToDNMapping and authzQueryTemplate parameter.  Doesn't have to be every test but at least one would be ideal.   Would that add a huge scope?

Reply:
Sure, will do that